nf_core/modules/remove.py

```diff
@@ -30,40 +31,23 @@ def remove(self, module):
         # Check whether pipelines is valid
         self.has_valid_directory()
 
-        # Get the installed modules
-        self.get_pipeline_modules()
-        if sum(map(len, self.module_names)) == 0:
-            log.error("No installed modules found in pipeline")
-            return False
-
-        # Decide from which repo the module was installed
-        # TODO Configure the prompt for repository name in a nice way
-        if True:
-            repo_name = self.modules_repo.fullname
-        elif len(self.module_names) == 1:
-            repo_name = list(self.module_names.keys())[0]
-        else:
-            repo_name = questionary.autocomplete(
-                "Repo name:", choices=self.module_names.keys(), style=nf_core.utils.nfcore_question_style
-            ).unsafe_ask()
-
+        repo_name = self.modules_repo.fullname
         if module is None:
             module = questionary.autocomplete(
-                "Tool name:", choices=self.module_names[repo_name], style=nf_core.utils.nfcore_question_style
+                "Tool name:",
+                choices=self.modules_from_repo(repo_name),
```

Comment:
I think this line should change in the other commands also, is that right?
For example the one from `patch.py` is failing

Reply:
Yes you are right, thanks! Though it seems like `patch.py` was the only one I missed, I've updated it now.